service/routes.go

```diff
@@ -95,6 +97,6 @@ func (svc *Service) privateEndpoints(ctx context.Context) {
 		svc.datasetAPIClient,
 	)
 	r.Get("/population-types/{population-type}/area-types/{area-type}/areas", areas.Get)
-
```

Comment:
Might be slightly clear to rename the existing areas.Get to areas.GetAll and have this as areas.Get. Also the url param could be called {area} rather than {area-id} to be consistent with the others.